app/src/main/java/com/simplecity/amp_library/ui/screens/folders/FolderFragment.java

```diff
@@ -227,10 +227,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
         recyclerView.setAdapter(adapter);
 
-        compositeDisposable.add(Aesthetic.get(getContext())
-                .colorPrimary()
-                .compose(distinctToMainThread())
-                .subscribe(color -> ViewBackgroundAction.create(appBarLayout).accept(color), onErrorLogAndRethrow()));
+//        compositeDisposable.add(Aesthetic.get(getContext())
+//                .colorPrimary()
+//                .compose(distinctToMainThread())
+//                .subscribe(color -> ViewBackgroundAction.create(appBarLayout).accept(color), onErrorLogAndRethrow()));
```

Comment:
`compositeDisposable.clear()` is still called later in the lifecycle, and other than that is completely unused. Could we remove entirely?